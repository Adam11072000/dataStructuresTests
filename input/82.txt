init 226
addImage -2
addImage -10
addImage -5
quit