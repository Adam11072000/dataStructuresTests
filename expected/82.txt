init done.
addImage: INVALID_INPUT
addImage: INVALID_INPUT
addImage: INVALID_INPUT
quit done.
